ArcGISRuntimeSDKQt_QMLSamples/Layers/BrowseOGCAPIFeatureService/BrowseOGCAPIFeatureService.qml

```diff
@@ -30,6 +30,7 @@ Rectangle {
     property OgcFeatureService featureService: null
     property FeatureLayer featureLayer : null
     property string errorMessage: ""
+    property bool loading: false
```

Comment:
I wonder if this could just be `property bool loading: featureService ? featureService.loadStatus === Enums.LoadStatusLoading : false`